### PR TITLE
Hubert / Small design fixes

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: registration_api.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ const appRegistrationMachine = createMachine({
                                     invoke: {
                                         // get the app id from the event
                                         src: async (_, event) => {
-                                            await removeApp(event.data);
+                                            await removeApp(event.data);;
```
Comment (adam-deriv) [inline registration_api.js:136]: why, removeeeve ;

File: registration_api.js [hunk 2]
```diff
@@ -573,12 +573,13 @@ const removeApp = async (app_id) => {
     send({ type: 'FETCH_APP_LIST' });
 }
 
-const appUpdate = async ({ app_id, name, redirect_uri, scopes }) => {
+const appUpdate = async ({ app_id, app_markup_percentage, name, redirect_uri, verification_uri, scopes }) => {
+    console.log('testestestestest');
```
Comment (adam-deriv) [inline registration_api.js:577]: whyyyyyyyy, remove pls

**Review by adam-deriv — CHANGES_REQUESTED** [review]

File: registration_api.js (hunk 2 above)
Comment (hubert-deriv) [reply registration_api.js:577]: Why, I think this looks amazing. JK.

File: registration_api.js (hunk 1 above)
Comment (hubert-deriv) [reply registration_api.js:136]: Fixed

**Review by adam-deriv — APPROVED** [review]

**Pull request merged**